Add pwd to show current directory

derekstavis opened the pull request:
Hello! Awesome project!

I noticed there's no `pwd` command. So I've implemented it.

This PR makes the root directory show as `/` and `cd` command accept `/` as well, as I noticed that `ls /public` and `ls /` already works.

I'm looking into implementing more commands 💯

zackargyle commented:
@derekstavis! Awesome, love it. Can you add a few tests to `tests/commands.js`? I'd love to see:

- should exist
- should print out cwd
- should print out '/' for empty path

Thanks again. You can add your name to the README if you'd like as well as the first `patron` :)

derekstavis commented:
@zackargyle Just implemented the tests, fixed some linter warnings and pushed the branch.

I'm unsure what to add to the Patrons section, in doubt I added "Be the second to contribute" as a quote to differentiate from the list. What should I add there? 😄

zackargyle commented:
lol I like it. Perfect.
